common/src/main/java/com/sequenceiq/cloudbreak/jerseyclient/client/WebTargetClientProvider.java

```diff
@@ -0,0 +1,18 @@
+package com.sequenceiq.cloudbreak.jerseyclient.client;
+
+import com.sequenceiq.cloudbreak.client.RestClientUtil;
+import org.springframework.stereotype.Component;
+
+import javax.ws.rs.client.Client;
+
+/**
+ * Simple component to make testing classes/methods easier which uses the javax.ws.rs.client.Client
+ */
+@Component
+public class WebTargetClientProvider {
```

Comment:
we have an injectable version with ssl and more shiny things, please talk with @keyki how you can use that stuff